.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 19
+          - 20
```

Comment:
Node 19 is now in maintenance mode, and Node 20 is out as the current version (will be promoted as active LTS on October 24, 2023).